Nikita Korzhavin/flex/cubes.css

```diff
@@ -0,0 +1,479 @@
+  src: local('AvenirNext-Bold'),
+    url('https://nomail.com.ua/files/woff/88da9ccecc9a384876ccf7c53eccec2d.woff')
+      format('woff');
+}
+
+body {
+  height: 100%;
+  min-width: 100%;
+  width: 100%;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+/*       виджет                */
+
+.bg1 {
+  width: 100%;
+}
+
+.bg2 {
+  width: 100%;
+}
+
+.widget {
+  width: 100%;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+/*           about cards               */
+.widget__card {
+  -webkit-box-flex: 1 1 100vw;
+  -ms-flex: 1 1 100vw;
+  flex: 1 1 100vw;
+}
+
+.card {
+  max-height: 84px;
+  font-size: 20px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -ms-flex-pack: distribute;
+  justify-content: space-around;
+}
+
+.card_flex {
+  -webkit-box-ordinal-group: 2;
+  -ms-flex-order: 1;
+  order: 1;
+  max-height: 80px;
+  padding: 0 10px;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.card__text {
+  color: #fff;
+  font-family: 'AvenirNext-Bold';
+}
+
+.card__text_size_m {
+  font-size: 1rem;
+}
+
+.card__img {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: end;
+  -ms-flex-pack: end;
+  justify-content: flex-end;
+}
+
+.card_bg-color_06b3db {
+  background-color: #06b3db;
+}
+
+.card_bg-color_4ed486 {
+  background: #4ed486;
+}
+
+.card_bg-color_6139f6 {
+  background: #6139f6;
+}
+
+.card_bg-color_e3b63d {
+  background: #e3b63d;
+}
+
+.card_bg-color_bd54cd {
+  background: #bd54cd;
+}
+
+.card_bg-color_bd3559 {
+  background: #bd3559;
+}
+
+.card__top {
+  display: none;
+}
+
+.card__bottom {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+}
+
+.card__info-wrapper {
+  width: 100%;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: space-between;
+}
+
+.card__info-day {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.card__text_size_s {
+  font-size: 0.814rem;
+}
+
+.card__img_size_s {
+  height: 30px;
+  min-width: 25px;
+}
+
+.card__img_opacity {
```

Comment:
I'd call it `opaque` since modifier name is rather adjective than noun.

Reply:
fixed